gettingStarted.md

```diff
@@ -0,0 +1,354 @@
+# Getting Started (for Developers)
+
+Welcome to POLAR! This guide will walk you through the initial steps of installation and development with POLAR, with a particular focus on using plugins.
+
+## 1. Overview
+
+POLAR is a versatile package that allows you to develop your own map projects using clients and plugins. This guide will show you how to use the Snowbox client and how to integrate plugins into your project.
```

Comment:
```suggestion
POLAR is a versatile package library that allows you to develop your own map projects using clients and plugins. This guide will show you how to use the Snowbox client and how to integrate plugins into your project.
```

Since it's a lot of packages, actually.

Reply:
done